modules/mobile/app/helpers/mobile/list_filter.rb

```diff
@@ -7,88 +7,84 @@ class ListFilter
     class FilterError < StandardError
     end
 
-    PERMITTED_OPERATIONS = %w[eq notEq].freeze
+    PERMITTED_OPERATIONS = %w[eq not_eq].freeze
```

Comment:
Changed the operation to snake case because that's what the Common::Collection filter logic uses and I thought it would be better to be consistent.